### DISCUSSION TITLE
安装完@tarojs/plugin-html这个后 在微信开发工具里面报错

### DISCUSSION THREAD

**Discussion by Buildvora** [NervJS/taro#11009] [Q & A]
![image](https://user-images.githubusercontent.com/43238325/147717216-a6903269-7409-4565-a79d-496edbfd03b2.png)
![image](https://user-images.githubusercontent.com/43238325/147717226-14bac3da-7f6e-46f0-9347-145083e43f93.png)
有没有大哥给说说是什么问题

**Comment by Chen-jj** [comment]
感觉是依赖装得有问题，重新安装试试
